wire/msgmixslotreserve.go

```diff
@@ -0,0 +1,319 @@
+// Copyright (c) 2023 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package wire
+
+import (
+	"fmt"
+	"hash"
+	"io"
+
+	"github.com/decred/dcrd/chaincfg/chainhash"
+)
+
+const (
+	// MaxMixMcount is the maximum number of mixed messages that are allowed
+	// in a single mix.  This restricts the total allowed size of the slot
+	// reservation and XOR DC-net matrices.
+	MaxMixMcount = 1024 // XXX: PNOOMA
+
+	// MaxMixKPCount is the maximum number of peers allowed together in a
+	// single mix.  This restricts the total size of the slot reservation
+	// and XOR DC-net matrices.
+	MaxMixKPCount = 512 // XXX: PNOOMA
```

Comment:
Reminder about this. Similar to the other cases, even though they might be PNOOMA, I'm sure there was a least some justification you used to arrive at these figures.

Rather than leaving an XXX, please document the rationale for the number even if it's something as simple as being based on a empirical data gathered from running the centralized mixing service plus some extra for additional wiggle room.